MastodonSDK/Package.resolved

```diff
@@ -72,15 +72,6 @@
         "version" : "4.2.2"
       }
     },
-    {
-      "identity" : "kingfisher",
-      "kind" : "remoteSourceControl",
-      "location" : "https://github.com/onevcat/Kingfisher.git",
-      "state" : {
-        "revision" : "44e891bdb61426a95e31492a67c7c0dfad1f87c5",
-        "version" : "7.4.1"
-      }
-    },
```

Comment:
These two changes (removing `kingfisher` and `thirdpartymailer` from `Package.resolved`) is just cleanup, right?

Reply:
Yes, I believe so — Xcode did this automatically.